### PR TITLE
[Quantum RPG] Hadamard Hills Rooms

### PR DESCRIPTION
**Pull request by dstrain115**
- Put in the room directions for the Hadamard Hills zone.
- Some descriptions are still pending.

### CODE REVIEW

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 1]
```diff
@@ -40,10 +41,241 @@
...
         items=[CONSTRUCTION_SIGN],
-        exits={Direction.SOUTH: "classical12"},
+        exits={Direction.SOUTH: "classical12", Direction.NORTH: "hadamard2"},
+    ),
+    Location(
+        label="hadamard2",
+        title="Bottom of Superposition Trail",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  There seems to be two ways up the slope that you can go up."
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:55]: REPLACE LINE
            "proper. There seem to be two ways that you can go up the slope."

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 2]
```diff
@@ -40,10 +41,241 @@
...
+            Direction.EAST: "hadamard3_0",
+            Direction.WEST: "hadamard3_1",
+        },
+    ),
+    Location(
+        label="hadamard3_0",
+        title="Bottom of Superposition Slope",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  The trails north leads steeply up to the top of the hills.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:69]: EITHER
{REPLACE trails
WITH trail},
{REPLACE leads
WITH lead}

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 3]
```diff
@@ -40,10 +41,241 @@
...
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard4_0", Direction.WEST: "hadamard2"},
+    ),
+    Location(
+        label="hadamard3_1",
+        title="Bottom of Superposition Slope",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  The trails north leads steeply up to the top of the hills.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:79]: Beware: at the moment, this is identical to location hadamard3_0.
I get the joke (superposition), so I know that this should be intentional. But, you know, doesn't hurt to flag it. If this is intentional, make the same edit as suggested in hadamard3_0.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 4]
```diff
@@ -40,10 +41,241 @@
...
+        title="Superposition Slope",
+        description=(
+            "You are climbing up a steep slope.  To the north is the Hadamard hills,\n"
+            "and to the south, the trail leads down to the river separating the\n"
+            "classical realm from the Hadamard Hills."
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard5", Direction.SOUTH: "hadamard3_0"},
+    ),
+    Location(
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:95]: Same comments on 3_0/3_1: 4_0 and 4_1 are identical. (Yep yep, superposition, I get it.)

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 5]
```diff
@@ -40,10 +41,241 @@
...
+            "classical realm from the Hadamard Hills."
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard5", Direction.SOUTH: "hadamard3_1"},
+    ),
+    Location(
+        label="hadamard5",
+        title="Top of Superposition Slope",
+        description=(
+            "You have reached the top of the superposition slope.  From here,\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:110]: CAPITALIZE superposition
Consider capitalizing "slope".

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 6]
```diff
@@ -40,10 +41,241 @@
...
+            "you can see the river winding its way through the valley from the south,\n"
+            "and the path downwards in that direction looks extremely steep and treacherous.\n"
+            "To the north, the foothills continue.  Two prominent peaks to the north-ease\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:113]: REPLACE ease
WITH east

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 7]
```diff
@@ -40,10 +41,241 @@
...
+            "and north-west are the most obvious features, though the land slopes gradually\n"
+            "towards the mountains of error correction in the northernmost distance.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:115]: Consider capitalizing "Mountains of Error Correction" if you plan for that to be their proper name later.

**Review by cognigami — CHANGES_REQUESTED** [review]

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 1 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:55]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 2 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:69]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 3 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:79]: Yes, it is by design.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 5 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:110]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 6 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:113]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 7 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:115]: Will pass on this one for now.  I am not sure what I will call them.  For now, it is a vague goal.

**Review by cognigami — APPROVED** [review]

**Pull request merged**